docs/deployment-on-pythonanywhere.rst

```diff
@@ -34,7 +34,7 @@ Make sure your project is fully committed and pushed up to Bitbucket or Github o
 
     git clone <my-repo-url>  # you can also use hg
     cd my-project-name
-    mkvirtualenv --python=/usr/bin/python3.10 my-project-name
+    mkvirtualenv --python=/usr/bin/python3.11 my-project-name
```

Comment:
It looks like Python anywhere doesn't support 3.11 yet? I don't see it mentioned in their documentation...